### PR TITLE
K8SPXC-213 Update default list of docker images which are used for tests

### PR DESCRIPTION
**Pull request by hors**

### CODE REVIEW

File: e2e-tests/functions [hunk 1]
```diff
@@ -7,10 +7,10 @@ GIT_COMMIT=$(git rev-parse HEAD)
 GIT_BRANCH=${VERSION:-$(git rev-parse --abbrev-ref HEAD | sed -e 's^/^-^g; s^[.]^-^g;' | tr '[:upper:]' '[:lower:]')}
 API="pxc.percona.com/v1-4-0"
 IMAGE=${IMAGE:-"perconalab/percona-xtradb-cluster-operator:${GIT_BRANCH}"}
-IMAGE_PXC=${IMAGE_PXC:-"perconalab/percona-xtradb-cluster-operator:master-pxc"}
+IMAGE_PXC=${IMAGE_PXC:-"perconalab/percona-xtradb-cluster-operator:master-pxc5.7"}
```
Comment (delgod) [inline e2e-tests/functions]:
```suggestion
IMAGE_PXC=${IMAGE_PXC:-"perconalab/percona-xtradb-cluster-operator:master-pxc8.0"}
```

File: e2e-tests/functions [hunk 2]
```diff
@@ -7,10 +7,10 @@ GIT_COMMIT=$(git rev-parse HEAD)
...
 IMAGE_PMM=${IMAGE_PMM:-"perconalab/percona-xtradb-cluster-operator:master-pmm"}
 IMAGE_PROXY=${IMAGE_PROXY:-"perconalab/percona-xtradb-cluster-operator:master-proxysql"}
-IMAGE_BACKUP=${IMAGE_BACKUP:-"perconalab/percona-xtradb-cluster-operator:master-backup"}
+IMAGE_BACKUP=${IMAGE_BACKUP:-"perconalab/percona-xtradb-cluster-operator:master-pxc5.7-backup"}
```
Comment (delgod) [inline e2e-tests/functions]:
```suggestion
IMAGE_BACKUP=${IMAGE_BACKUP:-"perconalab/percona-xtradb-cluster-operator:master-pxc8.0-backup"}
```

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | failed |
| self-healing | passed |
| big-data | passed |
| scaling | failed |
| recreate | passed |
| storage | failed |
| scaling-proxysql | passed |
| self-healing-advanced | failed |
| upgrade | passed |
| demand-backup | failed |
| operator-self-healing | passed |
| limits | passed |
| upgrade-consistency | passed |
| monitoring | failed |
| monitoring-2-0 | failed |
| scheduled-backup | failed |
| one-pod | passed |
| affinity | passed |
| proxysql-sidecar-res-limits | passed |
| auto-tuning | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| self-healing | failed |
| big-data | failed |
| recreate | failed |
| scaling | failed |
| init-deploy | failed |
| self-healing-advanced | failed |
| demand-backup | failed |
| scaling-proxysql | failed |
| storage | failed |
| operator-self-healing | failed |
| upgrade | passed |
| limits | failed |
| scheduled-backup | failed |
| upgrade-consistency | passed |
| one-pod | failed |
| monitoring | failed |
| auto-tuning | failed |
| monitoring-2-0 | failed |
| affinity | passed |
| proxysql-sidecar-res-limits | failed |

**Pull request merged**